web/assets/dark.scss

```diff
@@ -174,4 +180,11 @@ $sirius-bg-color: #29292e;
       fill: govuk-colour("white");
     }
   }
+
+  .govuk-inset-text,
+  .govuk-inset-text *,
+  .govuk-warning-text,
+  .govuk-warning-text * {
```

Comment:
Are the `*` selectors necessary? I'm conscious it might override things we don't want it to (a contrived example is that `<div class="govuk-inset-text"><span class="govuk-hint">Hint</span></div>` is white rather than the hint text colour).

Reply:
I could make it more specific to .gov-inset-text__text and similar, which would probably help, then make sure all places where we use the inset and warning components are correctly marked-up.